proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.audit.v1alpha1;
```

Comment:
Moved to its own separated package, just like it was on the previous implementation. It allowed me to remove all the `Audit*` prefixes. cc @robbertvanginkel @stefanvanburen

Reply:
probably we want to make this stable (`v1`) when we finish implementing the first list of the proposed audited events(?)

Reply:
> probably we want to make this stable (v1) when we finish implementing the first list of the proposed audited events(?)

on it's own, that sounds reasonable. 

though I think we should still double check to make sure there aren't any additional considerations before moving to `v1` (e.g. support level).

Reply:
yeah totally, that'd be at the end of the implementation of the current mvp